src/components/CustomCard/styles.module.scss

```diff
@@ -55,6 +58,7 @@
 }
 
 .imgSmall {
+    @include flex(space-between, flex-start, column);
     @include flex(space-between, flex-start, column);
```

Comment:
Same here and line 70, watch out for these mistakes!

Reply:
done